common/test_local_filestore.py

```diff
@@ -0,0 +1,58 @@
+# Copyright 2020 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""Tests for local_filestore.py."""
+
+from unittest import mock
+
+import pytest
+
+from common import local_filestore
+
+
+class TestLocalFileStoreRsync:
+    """Tests for local_filestore_command works as expected."""
```

Comment:
Will this cover only rsync, or the other commands as well? (Class name includes rsync, but docstring talks about `local_filestore_command`)

Reply:
This comment is outdated since we removed local_filestore_command now.